.github/workflows/ci.yaml

```
@@ -21,7 +19,7 @@ jobs:
   get-values:
     uses: ./.github/workflows/get-values.yaml
     permissions:
-      contents: write # needed updating dependabot branches
+      contents: write # needed for updating dependabot branches
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)